fli-iam/shanoir-ng
# shanoir-issue#2240: rename 'acquisition equipment' to 'center equipment'

How to test:
Open the center-equipment list and check there is a new column "modality"

The searches in this list and new column are working
The "acquisition equipment" was renamed to "center equipment" in breadcrumbs and in some fields (coils, study card, imports, etc)
